src/tally.F90

```diff
@@ -1160,8 +1160,13 @@ subroutine get_scoring_bins(p, i_tally, found_bin)
         coord => p % coord0
         offset = 0
         do while(associated(coord))
-          if (associated(coord % mapping)) then
-            offset = offset + coord % mapping(t % filters(i) % offset)
+          if (cells(coord % cell) % type == CELL_FILL) then
+            offset = offset + cells(coord % cell) % &
+                 offset(t % filters(i) % offset)
+          elseif(cells(coord%cell) % type == CELL_LATTICE) then
```

Comment:
Lack of whitespace in a few of these member accesses seems to be a bit inconsistent from what I've seen elsewhere in the source code - *e.g.*, ``coord%cell`` vs. ``coord % cell``. It doesn't look like this is mentioned in the style guide though, so I'm fine either way.